gitops/argocd/tektoncd/patch-tekton-config.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: patch-tekton-config-parameters
+  namespace: openshift-pipelines
+  annotations:
+    argocd.argoproj.io/hook: PostSync
+    argocd.argoproj.io/hook-delete-policy: HookSucceeded
+spec:
+  template:
+    spec:
+      containers:
+        - image: registry.redhat.io/openshift4/ose-cli:v4.9
+          command:
+            - /bin/bash
+            - -c
+            - |
+              echo "Waiting for TektonConfig config to be present"
+              until oc get TektonConfig config -n openshift-pipelines
+              do
+                sleep $SLEEP;
+              done
+
+              echo "Patching TektonConfig config patameters"
+              oc patch TektonConfig config --type='json' -p='[{"op":"remove","path":"/spec/pruner/keep"},{"op":"add","path":"/spec/pruner/keep-since","value":2880},{"op":"replace","path":"/spec/pruner/resources/0","value":"taskrun"},{"op":"replace","path":"/spec/pipeline/enable-tekton-oci-bundles","value":true}]'
```

Comment:
This is imperative. We don't need Argo CD if we patch in an imperative way one of the few config files we have